website/static/worker.js

```diff
@@ -111,8 +111,8 @@ function lazyLoadParser(parser) {
         : parser;
   var script = "parser-" + actualParser + ".js";
 
-  if (!parsersLoaded[actualParser]) {
+  if (!parsersLoaded[parser]) {
```

Comment:
This means it will call `importScripts` again if you try to load css after loading scss, for example. Is that a big deal?

I could change the check above in line 97 instead to get the actual parser, but this seemed simpler.

Reply:
How about checking the status in `lazyLoadParser()`? so that the `parsersLoaded` is only used in one place.

![image](https://user-images.githubusercontent.com/8341033/33109676-f25bab8e-cf7d-11e7-9297-77ac869175f3.png)